src/components/card/card.component.js

```diff
@@ -36,6 +37,7 @@ const Card = ({
       type='button'
       onClick={ onClickHandler }
       tabIndex={ 0 }
+      data-role={ dataRole || '' }
```

Comment:
It's better to conditionally add this prop otherwise we'll end up with `<div data-role></div>`